slirp4netns.c

```diff
@@ -8,6 +8,14 @@
 #include "libslirp.h"
 #include "api.h"
 
+typedef struct {
+	bool is_udp;
+	struct in_addr hostaddr;
+	uint16_t hostport;
+	struct in_addr guestaddr;
+	uint16_t guestport;
+} hostfwd_t;
+
```

Comment:
dup: slirp4netns.h

Reply:
With using the boilierplate from the api code this will be removed